# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipsec-tools/compile-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur ipsec-tools-0.6.1.orig/src/racoon/crypto_openssl.c ipsec-tools-0.6.1/src/racoon/crypto_openssl.c
--- ipsec-tools-0.6.1.orig/src/racoon/crypto_openssl.c	2005-07-12 14:50:15.000000000 +0300
+++ ipsec-tools-0.6.1/src/racoon/crypto_openssl.c	2005-09-16 01:23:58.000000000 +0300
@@ -81,6 +81,7 @@
 #ifdef HAVE_OPENSSL_SHA2_H
 #include <openssl/sha2.h>
 #else
+#define HAVE_SHA2_IN_SHA_H
 #include "crypto/sha2/sha2.h"
 #endif
 #endif
